frontend/src/app/modules/common/back-routing/back-routing.service.ts

```diff
@@ -50,7 +51,8 @@ export class BackRoutingService {
   public goBack(preferListOverSplit:boolean = false) {
     // Default: back to list
     // When coming from a deep link or a create form
-    const baseRoute = this.$state.current.data.baseRoute || 'work-packages.partitioned.list';
+    const baseRoute = this.backRoute.baseRoute || this.$state.current.data.baseRoute || 'work-packages.partitioned.list';
+    this.keepTab.baseRoute = baseRoute;
```

Comment:
I don't like this pattern (overriding a global service's parameters). Can't we just access the `baseRoute` from within the keepTab service?

Reply:
👍